### PR TITLE
DAP-2: Login design

### PR DESCRIPTION
**Pull request by Marcelkibor**
# Description of What I changed

Issue I worked on

see https://ampath.atlassian.net/browse/DAP-2

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+.yarn
+.env
```
Comment (kipchubett) [inline .gitignore]: Add a new line at the end of every file. Reason and how to fix it for once -> https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

File: vercel.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "rewrites":  [
+      {"source": "/(.*)", "destination": "/"}
+    ]
+  }
```
Comment (kipchubett) [inline vercel.json:5]: Ditto

File: src/authentication/Login/Login.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import Logo from '../../public/am-logo.jpg';
+
```
Comment (kipchubett) [inline src/authentication/Login/Login.tsx]: I'd recommend an SVG logo. I already shared it on slack; https://slack-files.com/T0YAREQ77-F04PJ0RQJSF-7525bc3f62

**Review by kipchubett — CHANGES_REQUESTED** [review]
Good progress @Marcelkibor !
Why are you deleting `vite.config.ts` file? Please go through the PR review changes

File: .gitignore (hunk 1 above)
Comment (Marcelkibor) [reply .gitignore]: Resolved

File: src/authentication/Login/Login.tsx (hunk 3 above)
Comment (Marcelkibor) [reply src/authentication/Login/Login.tsx]: Resolved

**Pull request closed without merging**